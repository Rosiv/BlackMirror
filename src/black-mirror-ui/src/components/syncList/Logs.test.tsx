import * as React from 'react';
import renderer from 'react-test-renderer';

import MuiThemeProvider from 'material-ui/styles/MuiThemeProvider';
import getMuiTheme from 'material-ui/styles/getMuiTheme';
import darkBaseTheme from 'material-ui/styles/baseThemes/darkBaseTheme';
const customMuiTheme = getMuiTheme(darkBaseTheme);
import Logs from './Logs';
var SyncData = require('../../test/sync.json');

it('renders and is compatible with the snapshot', () => {
    const tree = renderer
        .create(
        <MuiThemeProvider muiTheme={customMuiTheme}>
            <Logs sync={SyncData} />
        </MuiThemeProvider>)
        .toJSON();
    expect(tree).toMatchSnapshot();
});
