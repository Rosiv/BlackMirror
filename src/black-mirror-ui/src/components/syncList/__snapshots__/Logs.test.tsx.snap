// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders and is compatible with the snapshot 1`] = `
<div>
  <a
    disabled={false}
    href="/sync/F12/logs"
    onBlur={[Function]}
    onClick={[Function]}
    onFocus={[Function]}
    onKeyDown={[Function]}
    onKeyUp={[Function]}
    onMouseEnter={[Function]}
    onMouseLeave={[Function]}
    onTouchEnd={[Function]}
    onTouchStart={[Function]}
    open={null}
    style={
      Object {
        "WebkitTapHighlightColor": "rgba(0, 0, 0, 0)",
        "background": "none",
        "backgroundColor": null,
        "border": 10,
        "boxSizing": "border-box",
        "color": "rgba(255, 255, 255, 1)",
        "cursor": "pointer",
        "display": "block",
        "fontFamily": "Roboto, sans-serif",
        "fontSize": 16,
        "fontWeight": "inherit",
        "lineHeight": "16px",
        "margin": 0,
        "muiPrepared": true,
        "outline": "none",
        "padding": 0,
        "position": "relative",
        "textDecoration": "none",
        "transition": "all 450ms cubic-bezier(0.23, 1, 0.32, 1) 0ms",
        "verticalAlign": null,
      }
    }
    tabIndex={0}
  >
    <div
      onMouseDown={[Function]}
      onMouseLeave={[Function]}
      onMouseUp={[Function]}
      onTouchEnd={[Function]}
      onTouchStart={[Function]}
    >
      <div
        style={
          Object {
            "marginLeft": 18,
            "muiPrepared": true,
            "paddingBottom": 16,
            "paddingLeft": 72,
            "paddingRight": 16,
            "paddingTop": 16,
            "position": "relative",
          }
        }
      >
        <svg
          onMouseEnter={[Function]}
          onMouseLeave={[Function]}
          style={
            Object {
              "color": "rgba(255, 255, 255, 1)",
              "display": "block",
              "fill": "#bdbdbd",
              "height": 24,
              "left": 4,
              "margin": 12,
              "muiPrepared": true,
              "position": "absolute",
              "top": 0,
              "transition": "all 450ms cubic-bezier(0.23, 1, 0.32, 1) 0ms",
              "userSelect": "none",
              "width": 24,
            }
          }
          viewBox="0 0 24 24"
        >
          <path
            d="M18 17H6v-2h12v2zm0-4H6v-2h12v2zm0-4H6V7h12v2zM3 22l1.5-1.5L6 22l1.5-1.5L9 22l1.5-1.5L12 22l1.5-1.5L15 22l1.5-1.5L18 22l1.5-1.5L21 22V2l-1.5 1.5L18 2l-1.5 1.5L15 2l-1.5 1.5L12 2l-1.5 1.5L9 2 7.5 3.5 6 2 4.5 3.5 3 2v20z"
          />
        </svg>
        <div
          style={
            Object {
              "muiPrepared": true,
            }
          }
        >
          Logs
        </div>
      </div>
    </div>
  </a>
</div>
`;
